configuration/rules-dsl.md

```diff
@@ -90,19 +90,17 @@ The **Rules** section contains a list of rules.
 Each rule has the following syntax:
 
 ```java
-rule "rule name"
+rule "<RULE_NAME>"
 when
-    <TRIGGER CONDITION1> or
-    <TRIGGER_CONDITION2> or
-    <TRIGGER_CONDITION3>
-    ...
+    <TRIGGER_CONDITION>
 then
     <SCRIPT_BLOCK>
 end
 ```
 
-A rule can have any number of trigger conditions, but must at least have one.
-The `SCRIPT_BLOCK` contains the code that should be executed, when a trigger condition is met, see the [script](#scripts) section for details on its syntax.
+- `<RULE_NAME>` - Each rule must have a unique name (given within quotes). It is recommended to choose a meaningful description as rule name.
```

Comment:
Per the comment from @ThomDietrich, consider, "It is recommended that you choose a name that has meaning when spoken."